templates/projects/new.tmpl

```diff
@@ -18,7 +18,12 @@
 		</div>
 		<div class="field">
 			<label>{{ctx.Locale.Tr "repo.projects.description"}}</label>
-			<textarea name="content" placeholder="{{ctx.Locale.Tr "repo.projects.description_placeholder"}}">{{.content}}</textarea>
+			{{/* TODO: repo-level project and org-level project have different behaviros to render */}}
+			{{/* the "Repository" is nil when the project is org-level */}}
+			{{template "shared/combomarkdowneditor" (dict
+				"MarkdownPreviewInRepo" $.Repository "MarkdownPreviewContext" (Iif $.Repository "" .HomeLink) "MarkdownPreviewMode" (Iif $.Repository "comment")
+				"TextareaName" "content" "TextareaContent" .content "TextareaPlaceholder" (ctx.Locale.Tr "repo.projects.description_placeholder")
```

Comment:
One per line please

Reply:
Nope, I strongly prefer my style and it is carefully written.

One line for "markdown preview", one line for "textarea", and it isn't that long, the monitor screen should be width enough.

Reply:
I find one per line much more readable, espcially because the values are just stringed up with space between, no association between key and value visible.

Reply:
changed